data_collector/output.py

```diff
@@ -3,16 +3,15 @@
 import boto3
```

Comment:
Because we are actually uploading stuff to s3, I would recommend having that reflected in the file name.